ReactiveObjC/UITextField+RACSignalSupport.m

```diff
@@ -19,12 +19,15 @@ @implementation UITextField (RACSignalSupport)
 
 - (RACSignal *)rac_textSignal {
 	@weakify(self);
-	return [[[[[RACSignal
+	return [[[[[[RACSignal
 		defer:^{
 			@strongify(self);
 			return [RACSignal return:self];
 		}]
 		concat:[self rac_signalForControlEvents:UIControlEventAllEditingEvents]]
+		filter:^BOOL(UITextField *x) {
+			return !x.markedTextRange;
```

Comment:
Could a please add a comment to these explaining that they're necessary for multi-stage input in languages like Chinese?